### ISSUE TITLE
detecoRS, loading checkpoint loss go to nan

### ISSUE THREAD

**Issue by joihn** [open-mmlab/mmdetection#6271]
I have tried to train 
https://github.com/open-mmlab/mmdetection/tree/master/configs/detectors/detectors_cascade_rcnn_r50_1x_coco.py
on my custom dataset.

If I train from scratch, everything is fine.

However, If I load the proposed pre trained checkpoint:
https://download.openmmlab.com/mmdetection/v2.0/detectors/detectors_cascade_rcnn_r50_1x_coco/detectors_cascade_rcnn_r50_1x_coco-32a10ba0.pth

All my loss goes directly to Nan :(
Any idea why ?
 (I have already changeds the train pipeline with the mean/std of my custom dataset to normalize it correctly )

**Comment by joihn** [comment]
> @joihn did you remove the `semantic_head` in your model? I'm trying to do that, to train on my custom dataset, but I don't know how to do it:
> 
> #6250
> 
> Could you share your config file? It might be useful for me, even though I'm trying to use DetectoRS based on HTC, not based on Cascade R-CNN like you.

I think my model based on cascadercnn does not have a semantic head
![image](https://user-images.githubusercontent.com/11663917/137100511-ea99c86f-6958-4e7e-b069-7328bc1438d8.png)

**Comment by joihn** [comment]
> > I have already changed the train pipeline with the mean/std of my custom dataset to normalize it correctly
> 
> If the mean/std of your dataset and the COCO dataset are different, it is normal that you get the wrong results.

Well, we just have to adjust the harcoded mean/std of the data in the config file, 
so the new dataset get correctly  normalized by the train piepeline to std=1 and mean=0, right ?

**Comment by joihn** [comment]
> you're doing object detection, instead?
Exactly

**Comment by joihn** [comment]
> why did you define the whole model inside your config file, rather than importing a `_base_` config?

While not beeing the cleanest workflow, I find it easier for undertstanding what's happening in the cfg and modifying it.

> Did you use the same model defined in this _base_ config? https://github.com/open-mmlab/mmdetection/blob/master/configs/detectors/detectors_cascade_rcnn_r50_1x_coco.py

Originally yes but then I added some of my modifications. You can use a diff tool to see where are the modif.

**Comment by sam99dave** [comment]
> @joihn did you remove the `semantic_head` in your model? I'm trying to do that, to train on my custom dataset, but I don't know how to do it:
> 
> #6250
> 
> Could you share your config file? It might be useful for me, even though I'm trying to use DetectoRS based on HTC, not based on Cascade R-CNN like you.

I was able to train the detectors using the detectors_htc_r50_1x_coco.py config file with some changes. But the acc pumps to 100 and the losses become nan from the 200 iterations of 1st epoch.
Any idea for this behavior?

Config File - detectors_htc_r50_1x_coco.py ( Removed the stuffThingMap requirement )
pretrained = 'https://download.openmmlab.com/mmdetection/v2.0/detectors/detectors_htc_r50_1x_coco/detectors_htc_r50_1x_coco-329b1453.pth'
